molecule/commands.py

```diff
@@ -146,8 +144,12 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
         if self.molecule._state.get('converged'):
             create_inventory = False
 
+        if self.static:
+            create_instances = False
+            create_inventory = False
+
         if create_instances and not idempotent:
-            self.create()
+            Create(self.args, self.molecule).execute()
```

Comment:
Not a blocker, just my 2 cents:  
You used this pattern in a lot of places, but I prefer to see that split in 2 lines, one for creating the object, one for calling the function.